packages/demo-app-ts/src/Demo.css

```diff
@@ -77,4 +77,10 @@
 
 .pf-v6-c-toolbar__content.pf-m-hidden {
   display: none;
+}
+
+.pf-v6-c-page__main-container,
```

Comment:
I think we introduced a prop that applies these styles to page sections. Let me ask the scrum channel

Reply:
@jenny-s51 here's the response I got from @mcoker 

The general setup is, if you have a page section(s) you want to fill the available space, you add isContentFilled to <Page> and that tells the container that wraps the page content to fill the viewport (the white background will extend the height of the viewport). Then you need to add isFilled to any page sections you want to fill that space.

Here's an example w/ those deets - https://staging-v6.patternfly.org/components/page#filled-page-sections

and here's a codesandbox - https://codesandbox.io/s/sweet-frog-pp7zcr?file=/index.tsx:1761-1776

(FWIW the old behavior in v5 was that the main content area always filled the viewport height, and the last page section (whichever matches :last-child) would be filled by default. If you wanted to, for example, have the first page section fill the viewport instead, you'd add isFilled to the first page section, then add isFilled={false} to the last page section

If you see isFilled={false} that's probably why. That's no longer needed in v6)

Reply:
Removed the custom styling on the page components - added `hasBodyWrapper={false}` to the filled `<PageSections>` per @mcoker 's suggestion which resolved the issue